### ISSUE TITLE
Guidelines for advocacy for Coq and Coq-community

### ISSUE THREAD

**Issue by palmskog** [coq-community/manifesto#90]
Many Coq-community members regularly do advocacy for Coq itself, for Coq-community, type theory, machine-checked proofs, Coq libraries, Coq-verified software, and so on. It helps to have guidelines about how to best perform advocacy. This is just meant to start a discussion, but I can see at least the following (example) issues:

## Terminology
- Coq is a "proof assistant", or "interactive theorem prover" (not just a "theorem prover" or "program verification tool")
- Coq is based on "propositions-as-types", or "Curry-Howard correspondence", but perhaps not "Curry Howard isomorphism"(?) 

## Realism and Honesty
- Coq has had several [critical bugs](https://github.com/coq/coq/blob/master/dev/doc/critical-bugs), but theorems certified by Coq still have (much) higher trustworthiness than manual proofs, and [self-certification efforts](https://github.com/MetaCoq/MetaCoq) are underway
- There are many proof assistants with different foundations and capabilities, so a general ordering from "best to worst" should be avoided, but comparisons on concrete, objective, and user-relevant metrics are important for development progress.

The bottom line is that I think we should aim to include high-quality community-written advocacy guidelines in some form in the manifesto repository.

**Comment by spitters** [comment]
This initiative may be worth mentioning, and being expanded on:
https://popl20.sigplan.org/details/CoqPL-2020-papers/2/The-use-of-Coq-for-Common-Criteria-Evaluations

**Comment by palmskog** [comment]
One more note on terminology. The community has for some reason embraced "mechanization"/"mechanised"/"mechanical" [proofs] as a common keyword. I think it's much better to empasize **machine-checked** [proofs] instead, since it's bound to cause less confusion for outsiders.
